packages/sqs/lib/utils/sqsUtils.ts

```diff
@@ -107,8 +107,7 @@ export async function assertQueue(
         throw new Error('Queue ARN was not set')
       }
 
-      if (shallowEqual(existingAttributes.result, queueConfig.Attributes)) {
-      } else {
+      if (!isShallowSubset(existingAttributes.result, queueConfig.Attributes)) {
```

Comment:
is `shallowEqual` still used? I am not sure, just wondering if it could be removed

Reply:
good point, removed